src/utils/particle/updater.ts

```diff
@@ -63,13 +67,13 @@ export class Updater {
         const particle = this.particle;
 
         /* condensed particles */
-        let dx = particle.position.x - p2.position.x;
-        let dy = particle.position.y - p2.position.y;
-        let dist = Math.sqrt(dx * dx + dy * dy);
+        const dx = particle.position.x - p2.position.x;
+        const dy = particle.position.y - p2.position.y;
+        const dist = Math.sqrt(dx * dx + dy * dy);
 
         if (dist <= options.particles.line_linked.distance) {
-            let ax = dx / (options.particles.move.attract.rotateX * 1000);
-            let ay = dy / (options.particles.move.attract.rotateY * 1000);
+            const ax = dx / (options.particles.move.attract.rotateX * 1000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 1000](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)